api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -136,4 +136,6 @@ FirecloudWorkspaceACLUpdateResponseList updateWorkspaceACL(
 
   /** Creates a random Billing Project name. */
   String createBillingProjectName();
+
+  String getCompletedCloneWorkspaceFileTransfer(String workspaceNamespace, String firecloudName);
```

Comment:
We have a mix of "clone" and "duplicate" in the codebase but our preference is "duplicate"

Reply:
Yes i guess thats because firecloud has the attribute completedCloneWorkspaceFileTransfer. I can have our methods calling duplicate